launcher/modplatform/packwiz/Packwiz.cpp

```diff
@@ -72,7 +72,6 @@ auto stringEntry(toml::table table, QString entry_name) -> QString
 {
     auto node = table[StringUtils::toStdString(entry_name)];
     if (!node) {
-        qCritical() << "Failed to read str property '" + entry_name + "' in mod metadata.";
```

Comment:
qCritical()
> exits if the environment variable QT_FATAL_CRITICALS is not empty.
while I'm ok with changing it to a non fatal version I thing we should keep the log